go-controller/pkg/ovn/healthcheck/egressip_healthcheck.go

```diff
@@ -2,16 +2,16 @@ package healthcheck
 
 import (
 	"crypto/tls"
-	"net"
-	"strconv"
-	"sync"
-
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 	"golang.org/x/net/context"
+	"golang.org/x/net/proxy"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"k8s.io/klog/v2"
+	"net"
+	"strconv"
```

Comment:
nit: these should be grouped at the top like they were previously as they are part of stdlib